spec/fabricators/answer_fabricator.rb

```diff
@@ -1,2 +1,4 @@
 Fabricator(:answer) do
+  content { Faker::Lorem.words(3).join(' ') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.